### PR TITLE
Fix doc [#4939]

### LINKED ISSUE #4939: Fix some documentation typos and bad links
Fix some typos and formatting nits I've noticed as I've been working with Taskcluster docs. So far:

* https://jwhitlock.taskcluster-dev.net/docs/reference/workers/worker-runner/workers#docker-worker - Links to archived repo https://github.com/taskcluster/docker-worker, should link to https://github.com/taskcluster/taskcluster/tree/main/workers/docker-worker
* https://community-tc.services.mozilla.com/docs/reference/workers/worker-runner/workers#generic-worker - Whitespace in last paragraph on page, incorrect fixed text
* https://community-tc.services.mozilla.com/docs/manual/deploying#github - Link "Login Strategies" is broken, probably should go to https://community-tc.services.mozilla.com/docs/manual/deploying/ui#login-strategies
* https://github.com/taskcluster/taskcluster/blob/main/services/notify/README.md - "tess" should be "tests"
* https://www.npmjs.com/package/taskcluster-client-web - "You can install this package using Yarn **ore** npm"
* https://github.com/taskcluster/taskcluster/blob/main/dev-docs/dev-deployment.md - numbered list issue, wihch → which, etc.

It would be nice to run the all the documentation through a spell checker and link checker, but I'm just looking for small documentation fixes for this round.

### PR DESCRIPTION
**Pull request by JulienMaurice**
Github Bug/Issue: Fixes #4939 

Fixed the three first points of the issue as asked.

### CODE REVIEW

File: services/auth/README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ The auth service manages permissions and credentials in a Taskcluster deployment
 
 No special configuration is required for development.
 
-Run `yarn workspace taskcluster-hooks test` to run the tess.
+Run `yarn workspace taskcluster-hooks test` to run the test.
```
Comment (djmitche) [inline services/auth/README.md:9]: This should be plural -- there are more than one test :)

Comment (JulienMaurice) [reply services/auth/README.md]: Should be done now :)

**Review by petemoore — APPROVED** [review]
This is great, many thanks @JulienMaurice ! :-)

**Review by jwhitlock — APPROVED** [review]
Thanks @JulienMaurice, @djmitche and @petemoore!

**Pull request merged**